daemon/config/config.go

```diff
@@ -600,6 +609,64 @@ func findConfigurationConflicts(config map[string]interface{}, flags *pflag.Flag
 	return nil
 }
 
+// PreValidate verifies configs and handles conflicts in changed values
+func PreValidate(config *Config, configFile string, flags *pflag.FlagSet) (*Config, error) {
```

Comment:
https://github.com/moby/moby/pull/48151#discussion_r1687129847

Reply:
Addressed, hopefully the requirements don't change an 8th time.